eventhandlers/action_runners/actions/VertexTrainDatasetAction.py

```diff
@@ -0,0 +1,291 @@
+from eventhandlers.action_runners.base.ActionRunner import ActionRunner
```

Comment:
Hey Vitalii! Let's try adding the type hints to all the possible function params and variables here. That way we can get a better idea on what types of data each function is returning

Reply:
That comment is addressed to entire file, right?